Nanoleaf-wpf/Winealfs.External/Class1.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Winleafs.External
```

Comment:
Forgot to delete this

Reply:
Didn't forget. Rider makes the project completely different and much worse than VS does.
Fixed in next update